libkmod/libkmod.h

```diff
@@ -970,14 +970,32 @@ const char *kmod_module_get_path(const struct kmod_module *mod);
  */
 int kmod_module_get_dependency_symbols(const struct kmod_module *mod, struct kmod_list **list);
 
+/**
+ * kmod_symbol_bind:
+ * @KMOD_SYMBOL_NONE:
+ * @KMOD_SYMBOL_LOCAL:
+ * @KMOD_SYMBOL_GLOBAL:
+ * @KMOD_SYMBOL_WEAK:
+ * @KMOD_SYMBOL_UNDEF:
+ *
+ * The symbol bind type, see kmod_module_dependency_symbol_get_bind().
+ */
+enum kmod_symbol_bind {
+	KMOD_SYMBOL_NONE = '\0',
+	KMOD_SYMBOL_LOCAL = 'L',
+	KMOD_SYMBOL_GLOBAL = 'G',
+	KMOD_SYMBOL_WEAK = 'W',
+	KMOD_SYMBOL_UNDEF = 'U'
+};
```

Comment:
See kmod_symbol_bind_from_elf() and https://refspecs.linuxbase.org/elf/gabi4+/ch4.symtab.html

```
STB_LOCAL
Local symbols are not visible outside the object file containing their definition. Local symbols of the same name may exist in multiple files without interfering with each other.
STB_GLOBAL
Global symbols are visible to all object files being combined. One file's definition of a global symbol will satisfy another file's undefined reference to the same global symbol.
STB_WEAK
Weak symbols resemble global symbols, but their definitions have lower precedence.
```